build_fuzzer.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I don't see this file being used anywhere.  Does it really belong in the source tree?

If it does belong, could it go into the `scripts` subdirectory alongside the CI scripts?

Reply:
Not necessarily needed, but I thought it might be helpful if someone does not want to figure out the build options by themselves. Should it go into the scripts sub-directory or be completely removed?